plugins/minio.yaml

```diff
@@ -3,7 +3,7 @@ kind: Plugin
 metadata:
   name: minio
 spec:
-  version: v4.2.3
+  version: v4.2.4
```

Comment:
minio has since been upgraded to v4.2.5. Can you rebase on top of `origin/master` and make sure that the only changes come from your added file?